src/map/utils/battleutils.cpp

```diff
@@ -1429,20 +1429,18 @@ namespace battleutils
                 }
                 else if (daze == EFFECT_ASPIR_DAZE && power > 0 && PDefender->GetMaxMP() > 0)
                 {
-                    uint16 multiplier = 1 + (2 * power - 1);
+                    uint16 multiplier = (uint16)(1 + 2 * (power - 1));
```

Comment:
is the new uint16 here necessary?
`uint16 multiplier = 1 + 2 * (power - 1);` should work just as good and its easier to read

Reply:
Was matching the style above

Reply:
fixed